test/operations/placements/change_status_test.rb

```diff
@@ -0,0 +1,117 @@
+    actor_account = accounts(:admin_account)
+    old_status = placement.status
+    new_status = "no_reply"
+    reason = DisqualifyReason.find_by(title: new_status.humanize)
+
+    assert reason
+    assert_equal placement.status, "qualified"
+    assert_not placement.disqualify_reason
+    assert_not_equal old_status, new_status
+
+    assert_difference "Event.count" do
+      placement = Placements::ChangeStatus.new(
+        new_status:,
+        placement:,
+        actor_account:
+      ).call.value!
+
+      assert_equal placement.status, "disqualified"
+
+      assert_equal placement.disqualify_reason, reason
+      assert_equal reason.title, new_status.humanize
+      assert_equal reason.description, disqualify_reasons(:no_reply_toughbyte).description
+
+      event = Event.last
+
+      assert_equal event.actor_account_id, actor_account.id
+      assert_equal event.type, "placement_changed"
+      assert_equal event.eventable_id, placement.id
+      assert_equal event.eventable_type, "Placement"
+      assert_equal event.changed_field, "status"
+      assert_equal event.changed_from, old_status
+      assert_equal event.changed_to, new_status
+      assert_equal event.properties["reason"], new_status.humanize
+    end
+  end
+
+  test "should change status, create event and not assign disqualify_reason " \
+       "if status changed to reserved or qualified" do
+    placement = placements(:sam_golang_replied)
+    actor_account = accounts(:admin_account)
+    old_status = placement.status
+    new_status = "reserved"
+    reason = DisqualifyReason.find_by(title: new_status.humanize)
+
+    assert_not reason
+    assert_equal placement.status, "qualified"
+    assert_not placement.disqualify_reason
+    assert_not_equal old_status, new_status
+
+    assert_difference "Event.count" do
+      placement = Placements::ChangeStatus.new(
+        new_status:,
+        placement:,
+        actor_account:
+      ).call.value!
+
+      assert_equal placement.status, "reserved"
+      assert_not placement.disqualify_reason
+
+      event = Event.last
+
+      assert_equal event.actor_account_id, actor_account.id
+      assert_equal event.type, "placement_changed"
+      assert_equal event.eventable_id, placement.id
+      assert_equal event.eventable_type, "Placement"
+      assert_equal event.changed_field, "status"
+      assert_equal event.changed_from, old_status
+      assert_equal event.changed_to, new_status
+      assert_empty event.properties
+    end
+  end
+
+  test "should change status, create event and unassign disqualify_reason " \
+       "if status changed from disqualified to qualified" do
+    placement = placements(:sam_golang_sourced)
+    actor_account = accounts(:admin_account)
+    old_status = placement.status
+    new_status = "qualified"
+
+    assert_equal placement.status, "disqualified"
+    assert placement.disqualify_reason
+    assert_not_equal old_status, new_status
+
+    assert_difference "Event.count" do
```

Comment:
why do you end block assert after all other asserts? In most tests we ends it right after calling operation that should create it